runtime/oti/vmconstantpool.xml

```diff
@@ -411,13 +407,15 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<staticmethodref class="java/lang/J9VMInternals" name="initializationAlreadyFailed" signature="(Ljava/lang/Class;)V"/>
 	<staticmethodref class="java/lang/J9VMInternals" name="recordInitializationFailure" signature="(Ljava/lang/Class;Ljava/lang/Throwable;)V"/>
 
+	<!-- Static method references needed to support OpenJ9 and OpenJDK MethodHandles. -->
+	<staticmethodref class="java/lang/invoke/MethodHandleHelper" name="sendResolveMethodHandle" signature="(ILjava/lang/Class;Ljava/lang/Class;Ljava/lang/String;Ljava/lang/String;Ljava/lang/ClassLoader;)Ljava/lang/invoke/MethodHandle;" flags="opt_methodHandleCommon"/>
+	<staticmethodref class="java/lang/invoke/MethodHandleHelper" name="resolveInvokeDynamic" signature="(JLjava/lang/String;Ljava/lang/String;J)Ljava/lang/invoke/MethodHandle;" flags="opt_methodHandleCommon"/>
+	<staticmethodref class="java/lang/invoke/MethodHandleHelper" name="resolveConstantDynamic" signature="(JLjava/lang/String;Ljava/lang/String;J)Ljava/lang/Object;" versions="11-" flag="opt_methodHandleCommon"/>
+	<staticmethodref class="java/lang/invoke/MethodHandleHelper" name="constructorPlaceHolder" signature="(Ljava/lang/Object;)Ljava/lang/Object;" flags="opt_methodHandleCommon"/>
```

Comment:
It's not needed for OJDK MethodHandles. But, it is a static method referred in vmconstantpool.xml. Leaving it in MethodHandle will cause JCL init issues https://github.com/eclipse/openj9/pull/10637#issuecomment-696193146 since it won't exist in OJDK MethodHandle. So, I decided to remove all static methods, which are referred in vmconstantpool.xml, out of MethodHandle into MethodHandleHelper. 

There is one static method (referred in vmcp.xml) pending to be moved: `MH.invokeWithArgumentsHelper`. It will be handled in a separate PR since it has a lot of JIT usages in OpenJ9 and OMR.